default.hbs

```diff
@@ -12,19 +12,22 @@
     <meta name="HandheldFriendly" content="True" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    <link rel="stylesheet" type="text/css" href="https://cdn.freecodecamp.org/news-assets/prism-1-16-0/prism.min.css" />
-    <link rel="stylesheet" type="text/css"
+    <link rel="preload" as="style" onload="this.onload=null;this.rel='stylesheet'" href="https://cdn.freecodecamp.org/news-assets/prism-1-16-0/prism.min.css" />
+    <noscript><link rel="stylesheet" href="https://cdn.freecodecamp.org/news-assets/prism-1-16-0/prism.min.css"></noscript>
+    <link rel="preload" as="style" onload="this.onload=null;this.rel='stylesheet'"
         href="https://cdn.freecodecamp.org/news-assets/prism-1-16-0/prism-unescaped-markup.css" />
+    <noscript><link rel="stylesheet" href="https://cdn.freecodecamp.org/news-assets/prism-1-16-0/prism-unescaped-markup.css"></noscript>
 
-    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,400i,700|Roboto+Mono:400,700" rel="stylesheet">
+    <link rel="preload" as="font" crossorigin href="https://fonts.googleapis.com/css?family=Lato:300,400,400i,700|Roboto+Mono:400,700" rel="stylesheet">
```

Comment:
This seems to get rid of the `The resource ... was preloaded using link preload but not used within a few seconds from the window's load event` warning:

```suggestion
    <link rel="preload stylesheet" as="style" crossorigin href="https://fonts.googleapis.com/css?family=Lato:300,400,400i,700|Roboto+Mono:400,700" />
```

Reply:
Oh, wow, I had two `rel`s :facepalm:

I *think* we need to just use "preload", though, like this https://stackoverflow.com/questions/61889044/how-to-use-both-rel-preload-and-rel-stylesheet-for-the-same-tag suggests.  Also, I'll need to check on Firefox to see if the preload issue has been solved.

Reply:
Awesome, thanks for catching that. I didn't think to check for browser differences with preload. Yeah, let's try to figure out one of the methods in that thread that works with all browsers and possibly a fallback.